Tests/test_PDB_NeighborSearch.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 by Peter Cock.  All rights reserved.
```

Comment:
Sorry, my mistake - I copy-pasted this but meant to put your name here.

Also I think it would be reasonable to dual license this file, so:


```
# Copyright 2017 by Sourav Singh.  All rights reserved.
# Based on code Copyright 2002 by Thomas Hamelryck.  All rights reserved.
# This file is part of the Biopython distribution and governed by your
# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
# Please see the LICENSE file that should have been included as part of this
# package.
```